src/phoenix/core/model_schema.py

```diff
@@ -800,7 +800,11 @@ def __init__(
         # features and tags.
         for dim in self[ScalarDimension]:
             if dim.data_type is UNKNOWN:
-                data_type = _guess_data_type(map(dim, self._datasets.values()))
+                phisical_series = []
```

Comment:
did you mean physical?

Reply:
thanks for catching the typo!

`virtual` always sounds a bit hoity-toity to me, but maybe that's just me. haha